File: 0229-majority-element-ii/0229-majority-element-ii.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var majorityElement = function(nums) {
    nums.sort();
    let counter = 0;
    let ans = [];
    let element = -10000000000;
    for(let i = 0; i < nums.length; i++)
        {
            for(let j = i; j < nums.length; j++)
                {
                    if(nums[i] == nums[j])
                        {
                            counter++;
                        }
                }
            if(counter > (nums.length / 3) && element !== nums[i])
                {
                    ans.push(nums[i]);
                    element = nums[i];
                }
            counter = 0;
        }
    return ans;
};

File: 0229-majority-element-ii/0229-majority-element-ii.cs
public class Solution {
    public IList<int> MajorityElement(int[] nums) {
    var dic = new Dictionary<int, int>();
for(var i = 0; i < nums.Length; i++)
{
    if (!dic.ContainsKey(nums[i]))
        dic.Add(nums[i], 1);
    else
        dic[nums[i]]++;
}
var result = (from element in dic
             where element.Value > nums.Length / 3
             select element.Key).ToList<int>();

    return result;
    }
}